db/seeds.rb

```diff
@@ -14,23 +14,24 @@
 Item.create(name: 'Beer', description: 'Proof God wants us to be happy', price: 3.99, category_id: 2, image_url: "http://cdn.thedailybeast.com/content/dailybeast/articles/2014/04/05/wine-snobs-there-s-a-beer-for-you/jcr:content/image.crop.800.500.jpg/46659654.cached.jpg")
 Item.create(name: 'Milk', description: 'Grow big and strong', price: 1.99, category_id: 2, image_url: "http://nohealthnolife.net/wp-content/uploads/2015/07/1111.jpg")
 Item.create(name: 'Fries', description: 'Now with no trans fat!', price: 1.99, category_id: 3, image_url: "http://www.foodadvisor.my/attachments/store/fill/800/500/8ab6f5150b3b9b4d70be356908e574932938da53fd55a45a379b593fc649/featured_image.jpg")
-Item.create(name: 'Green Beans', description: 'Good and good for you!', price: 2.99, category_id: 3, image_url: "http://postcards.thepurplepassport.com/wp-content/uploads/2013/07/Green-Beans-with-Pork-at-Din-Tai-Fung_7026358825_l-e1372871437478.jpg")
-Item.create(name: 'Onion Rings', description: 'Deep fried, deep pleasure', price: 2.99, category_id: 3, image_url: "https://lh3.googleusercontent.com/-xT2o5NUey3E/VY1JxkEtufI/AAAAAAAABkE/_hAiOPqRL0A/w800-h800/Spicy_Indian_Style_Onion_Rings_Recipe_thumbnail_1280x800.jpg")
+item = Item.create(name: 'Green Beans', description: 'Good and good for you!', price: 2.99, category_id: 3, image_url: "http://postcards.thepurplepassport.com/wp-content/uploads/2013/07/Green-Beans-with-Pork-at-Din-Tai-Fung_7026358825_l-e1372871437478.jpg")
+item2 = Item.create(name: 'Onion Rings', description: 'Deep fried, deep pleasure', price: 2.99, category_id: 3, image_url: "https://lh3.googleusercontent.com/-xT2o5NUey3E/VY1JxkEtufI/AAAAAAAABkE/_hAiOPqRL0A/w800-h800/Spicy_Indian_Style_Onion_Rings_Recipe_thumbnail_1280x800.jpg")
```

Comment:
Line is too long. [279/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.